pom.xml

```diff
@@ -13,8 +13,8 @@
 
   <parent>
     <groupId>com.google.cloud</groupId>
-    <artifactId>google-cloud-shared-config</artifactId>
-    <version>1.7.1</version>
+    <artifactId>sdk-platform-java-config</artifactId>
+    <version>3.23.0</version>
```

Comment:
should we also make this a variable/property for this and the above usage?

Reply:
Thanks for the suggestion! I was wondering that too since we have the version specified in two places but unfortunately I don't know if it possible to use a placeholder for the parent since attempting to do so results in this error message: 
```
Properties in parent definition are prohibited
```

I'm guessing Maven tries to determine the parent-child relationships first and then performs property interpolation. Open to any other ideas though.

Reply:
ah okay, no worries - thanks for the explanation!